### DISCUSSION TITLE
Can't apply for Student Developer Pack because my University offers neither dated IDs nor sends letters printed in paper

### DISCUSSION THREAD

**Discussion by lynatic1337** [community/community#57875] [GitHub Education]
### Hi everyone,

Hey, I'm trying to apply to the Student Developer pack. Unfortunately my university only offers undated IDs. Because of an environment effort, they don't send us out letters in paper but allow us to download them on their online portal.

Now some genius at GitHub made it a policy to reject any proof that isn't a photo of a printed document because "digital documents aren't as trustworthy [...]". This not only is extremely dumb because it essentially differentiates between fraudsters and non fraudsters by the ownership of a printer, it's also a punch in the face to any person trying to make an environmental effort.

(It's also an unnecessary measure because our universities digital document literally contain a QR code to verify the legitimacy of the documents on the universities website.)

So how do I apply now? I don't own a printer and even if I did, by printing out this letter I'd essentially commit document forgery because this wouldn't be an official letter but only a copy.

### A suggestion...(optional)

An application form that is built to support edge cases and the ability to appeal decision that aren't even made by a human.

**Comment by iirfrankingindia** [comment]
Indian universities are regarded as the best in the world by students from all over, largely due to a variety of factors, including the country's diversely rich culture, technological advancements, wide range of courses, well-equipped labs, and affordable cost of living. India's universities can be broadly categorized as either private (self-financed) or public (government-funded). A university can differentiate itself from others through a number of different factors working together. Some of the most important prerequisites that are crucial in building a positive reputation for the university include well-trained faculty, a strong record of placement, and strong infrastructure.

The IITs and IIMs are renowned as excellence centers. A multidisciplinary educational system has recently been introduced in India through the establishment of numerous liberal arts universities. It does not just give students a different option from Ivy League institutions; it also demonstrates the bright future of liberal arts education in India. The IIRF University Ranking 2023 offers a list of the [best universities in India.

**Comment by samus-aran** [comment]
Hi @AgentHV 👋 

First off I want to thank you for sharing your experiences and that we absolutely understand your frustration. We are trying our best to continually improve our system to be as easy as possible for legitimate students everywhere to verify their academic status, while also ensuring that bad actors impersonating students don't jeopardise our ability to provide academic benefits of value.

As you can imagine there are many types of academic verification and our system is constantly evolving to accommodate this without setting the bar too high. However sometimes we may require a little more evidence from the student applying and we thank you for being patient.

It would be preferable that a student is on campus when applying and thus able to upload a document but with remote learning that isn't always possible, so we request students submit a live photo of their evidence rather than an upload. We understand that students may not have access to high-quality cameras (laptop or phones) but as long as the text is legible the system should be able to process it. Here are a few tips on using a photo to capture.

1. Ensure your image (photo or upload) is legible. If you are unable to read its text on the image itself then the verification process may not be able to either. We do accept live images of online documents. If you are taking a photo of a screen try to have it as large a resolution as possible - **take a photo of the evidence on a monitor (a laptop perhaps - or a TV) rather than a phone.**
2. If necessary, print off your evidence and take a photo of this copy.
3. Keep the text legible. It's preferred to have the full shot of the evidence but try and capture the important sections of it to confirm your student status. If the photo is too large and makes the text tiny then zoom in appropriately.

I would also suggest to be explicit about the feedback given to you from the rejected application submission. Do you have a screenshot of the rejection reasons? There are varying factors in the verification process and although we do prefer live images of evidence I suspect there are more important issues with your application that need to be resolved.

Hope this helps 👍

**Reply by lynatic1337** [reply]
<img width="660" alt="image" src="https://github.com/community/community/assets/39234676/a7e15f95-b9e3-4055-a690-7dc1201c2c5d">

I got this error multiple times, it always happened. I can pm you the document if that would help improve the process. 

Please note that my university is not a remote learning university but still only makes documents available via an online platform to their students on campus. This is due to environmental concerns. You point 2 defeats those. There is literally no way for me to get that document in physical form other than borrowing a printer and printing it out (which I eventually did).

If I may add another comment, requiring that the document is in paper-form is pointless. In the crazy world of 2023 we live in, fraudsters probably own printers too. This literally achieves nothing except forcing every legit student to waste power, ink and a piece of paper.

**Reply by samus-aran** [reply]
Thanks @AgentHV 

> Please note that my university is not a remote learning university but still only makes documents available via an online platform to their students on campus. This is due to environmental concerns. You point 2 defeats those. There is literally no way for me to get that document in physical form other than borrowing a printer and printing it out (which I eventually did).

We don't require users to print documents, we accept images taken of online documents. What we do highlight is we prefer users not to upload or use a file picker to submit their documents. So we request students submit a live photo of their evidence.

We see that you have now received your verified student status 🎉  We appreciate you responding and offering to help improve the process. I suspect a step was missed when your profile was updated which was to log out and log back in again before re-applying. This may have affected you, which has since been fixed and so should not affect other users.

Again, thanks for coming back and replying. Enjoy your GitHub Global Campus benefits 👍

**Comment by AndrewPopesku** [comment]
It seems like I have the same problem. I have the ID card with dates, but It is written in Ukrainian.

Could you help me please?
